.NET/Microsoft.Recognizers.Text.NumberWithUnit/Korean/Extractors/KoreanNumberWithUnitExtractorConfiguration.cs

```diff
@@ -94,7 +94,9 @@ public void ExpandHalfSuffix(string source, ref List<ExtractResult> result, IOrd
                         var match_suffix = new List<ExtractResult>();
                         foreach (var mr in match)
                         {
-                            if (mr.Start == (start + length))
+                            var subLength = (int)mr.Start - (start + length) >= 0 ? (int)mr.Start - (start + length) : 0;
```

Comment:
Can you add a code comment here to explain this behaviour difference?

Reply:
added (needed to take into account whitespaces between result and half unit).